frontend/src/components/metadata/EditMetadataDefinitionModal.tsx

```diff
@@ -0,0 +1,671 @@
+										arrow
+									>
+										<Grid container alignItems="center">
+											<Typography
+												variant="subtitle1"
+												style={{ marginRight: "10px" }}
+											>
+												Required:
+											</Typography>
+											<Grid item sm={6} md={4}>
+												<FormControlLabel
+													control={
+														<Checkbox
+															checked={formInput.required_for_items.datasets}
+															onChange={() =>
+																handleMetadataRequiredInputChange("datasets")
+															}
+														/>
+													}
+													label="Datasets"
+												/>
+												<FormControlLabel
+													control={
+														<Checkbox
+															checked={formInput.required_for_items.files}
+															onChange={() =>
+																handleMetadataRequiredInputChange("files")
+															}
+														/>
+													}
+													label="Files"
+												/>
+											</Grid>
+										</Grid>
+									</Tooltip>
+								</FormGroup>
+
+								{contextMap.map((item, idx) => {
+									return (
+										<Grid container>
+											<Grid item>
+												<ClowderInput
+													margin="normal"
+													fullWidth
+													required
+													id="metadata-context"
+													label="Term"
+													placeholder="Please enter context term"
+													value={item["term"]}
+													sx={{
+														mt: 1,
+														mr: 1,
+														alignItems: "right",
+														width: "300px",
+													}}
+													onChange={(event) => {
+														updateContext(idx, "term", event.target.value);
+													}}
+												/>
+											</Grid>
+											<Grid item>
+												<Autocomplete
+													id="metadata-auto-complete"
+													freeSolo
+													autoHighlight
+													inputValue={item["iri"]}
+													options={contextUrlMap["frequently_used"].map(
+														(option) => option.url
+													)}
+													onInputChange={(event, value) => {
+														updateContext(idx, "iri", value);
+													}}
+													renderInput={(params) => (
+														<ClowderInput
+															{...params}
+															required
+															label="IRI"
+															fullWidth
+														/>
+													)}
+													sx={{
+														mt: 1,
+														ml: 2,
+														alignItems: "right",
+														minWidth: "20em",
+													}}
+												/>
+											</Grid>
+											<IconButton
+												color="primary"
+												size="small"
+												onClick={() => addNewContext(idx)}
+											>
+												<AddBoxIcon />
+											</IconButton>
+											{idx == 0 ? (
+												<></>
+											) : (
+												<IconButton
+													color="primary"
+													size="small"
+													onClick={() => removeContext(idx)}
+												>
+													<DeleteOutlineIcon />
+												</IconButton>
```

Comment:
<img width="732" alt="image" src="https://github.com/clowder-framework/clowder2/assets/13950475/562abdbb-de8e-4368-a678-44ef50659683">

The first item cannot be deleted in this case. You might want to relax the `idx == 0 ?` criterion

Reply:
Done.